src/Float16.h

```diff
@@ -16,43 +15,19 @@ namespace Halide {
  *  to code that checks a type's size and used for buffer_t allocation.
  * */
 struct float16_t {
-    // NOTE: Do not use virtual methods here
-    // it will change the size of this data type.
 
     /// \name Constructors
     /// @{
 
-    /** Construct from a float using a particular rounding mode.
+    /** Construct from a float, double, or int using a particular rounding mode.
```

Comment:
Is "using a particular rounding mode" still true? If the input is expected to be exact anyway, does the rounding mode matter?